sdks/python/apache_beam/io/filebasedsink.py

```diff
@@ -15,7 +15,10 @@
 # limitations under the License.
 #
 
-"""File-based sources and sinks."""
+"""File-based sources and sinks.
+
+This module is deprecated, no backwards-compatibility guarantees.
```

Comment:
This module is not deprecated. We should prob. mark iobase.Sink and iobase.Writer as deprecated since those were removed from Java SDK.